wsmaster/codenvy-hosted-api-license/src/main/java/com/codenvy/api/license/server/LicenseService.java

```diff
@@ -199,4 +212,33 @@ public Response isCodenvyUsageLegal() throws ApiException {
         }
     }
 
+    @GET
+    @Path("/legality/node")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Check if Codenvy node usage matches Codenvy License constraints, or free usage rules. If parameter nodeNumber " +
+                          "is required, then it will be validated, otherwise number swarm nodes will be validated")
```

Comment:
For me, required MUST be setted
optional should be missed
> It could have a value in case of requesting from Add Node IM CLI command.

Yes, it could have a value or It could have no a value. For me it is optional
No?